mim/commands/download.py

```diff
@@ -152,3 +187,60 @@ def download(package: str,
                     highlighted_error(f'{config_path} is not found.'))
 
     return checkpoints
+
+
+def _download_dataset(package: str, dataset: str, dest_root: str) -> None:
+    if platform.system() != 'Linux':
+        raise RuntimeError('download dataset is only available for Linux!')
+
+    if not is_installed(package):
+        raise RuntimeError(f'Please install {package} by ')
+
+    installed_path = get_installed_path(package)
+    mim_path = osp.join(installed_path, '.mim')
+    dataset_index_path = osp.join(mim_path, 'dataset-index.yml')
+
+    if not osp.exists(dataset_index_path):
+        raise FileNotFoundError(
+            f'Cannot find dataset-index.yaml in {dataset_index_path}, '
```

Comment:
```suggestion
            f'Cannot find dataset-index.yaml in {mim_path}, '
```

Reply:
Refine to f'Cannot find {dataset_index_path}'. The full path in error message could help users to locate the file easier.